src/calculateRentalCost.js

```diff
@@ -4,7 +4,19 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const RENTALCOST = 40 * days;
+  const SHORT_DAYS = 3;
+  const LONG_DAYS = 7;
+
+  if (days < SHORT_DAYS) {
+    return RENTALCOST;
+  }
+
+  if (days < LONG_DAYS) {
+    return RENTALCOST - 20;
```

Comment:
the discounts could also be added as consts, if you may change this, it would be an even cleaner code and more modifiable

Reply:
Ok ill change this, thx bro